src/index.html

```diff
@@ -46,6 +46,12 @@
     <!-- <script src="https://cdn.rawgit.com/robflaherty/riveted/master/riveted.min.js"></script>
     <script>riveted.init();</script> -->
     <!-- End Google Tag Manager -->
+    <!-- Web socket connection -->
+    <script src="http://eb-ci.wmm63vqska.eu-west-1.elasticbeanstalk.com/primus/primus.js"></script>
```

Comment:
This can be put into S3 at some point, the only requirement is that it matches the version of primus that is currently deployed. This endpoint is purely for development.